lambda_src/drivers/process_sentry_logger.py

```diff
@@ -95,6 +96,8 @@ def process_row(
         name,
         history_type
     )
+    CONSOLE_LOGGER.info(f'Constructed history_url: {history_url}.')
+    status: Optional[str] = None
```

Comment:
seems this is always returned, so you can just return it instead of creating an optional that you set once

Reply:
Makes sense.